api/src/games/prizes.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+class Prize:
```

Comment:
Call this class PrizeManager, PrizePool or something similar and create a new class to describe a singular prize.